src/main/java/org/project36/qualopt/web/rest/StudyResource.java

```diff
@@ -165,12 +167,14 @@ else if (authentication.getPrincipal() instanceof String) {
     @PostMapping(path = "/studies/send",
         produces={MediaType.APPLICATION_JSON_VALUE, MediaType.TEXT_PLAIN_VALUE})
     @Timed
-    public ResponseEntity sendStudy(@Valid @RequestBody Study study) {
-        log.debug("REST request to send Study : {}", study);
+    public ResponseEntity sendStudy(@Valid @RequestBody Invitation inv) {
+        Study study = inv.getStudy();
+        int delay = Integer.parseInt(inv.getDelay())/1000;
```

Comment:
is there a specific reason why delay is of type string but not int? (maybe provide a comment if this is a work around for something?)

Reply:
Not too sure about why inv.getDelay() returns a string that needs to be parsed?

Reply:
thank you for pointing that out, changed to int